dss/api/files.py

```diff
@@ -110,9 +114,18 @@ def get_helper(uuid: str, replica: Replica, version: str = None, token: str = No
                     path=get_dst_key(blob_path)
                 )))
             else:
+                # TODO: Add this into the cloud blobstore
+                filename = determine_filename(parent_bundle_fqid, file_fqid) or blob_path[len('blobs/'):]
```

Comment:
Handle case where filename is None. You don't want to set the CD header if user doesn't pass in a bundle fqid. Right now you'd be setting the header to a file name of `"None"` which is not right.

Reply:
It should be handling it, it's defaulting to `blob_path[len('blobs/'):]` (your current default in the browser according to #1837) when `parent_bundle_fqid == None`.

Reply:
Huh? If `parent_bundle_fqid` is `None`, `determine_filename` will return `None` and `filename` will be set to `None`. Three lines down, `filename` is interpolated into the CD header, causing `kwargs` to be set to `{'ResponseContentDisposition': f'attachment; filename=None'}`.

Reply:
The line is:
`filename = determine_filename(parent_bundle_fqid, file_fqid) or blob_path[len('blobs/'):]`

You need to scroll right.

Reply:
I see. So I was wrong. But does it give you pause that a I didn't pick up on this? Why not just 

```suggestion
                if parent_bundle_fqid:
                    filename = determine_filename(parent_bundle_fqid, file_fqid)
                else:
                    filename = blob_path[len('blobs/'):]
```